### ISSUE TITLE
score with  staff beginning after main ones or ending before main ones

### ISSUE THREAD

**Issue by phdm** [Audiveris/audiveris#632]
Hello Hervé,

first of all, kudos for this wonderful software.

I have a (I hope) small problem.  I have music scores from songs with a long instrumental intro and/or outro, where there is no staff for the voice score until the singer really begins to sing and after the singer stopped singing.  I did not find in the documentation how I can explain to audiveris that staffs 1,2,3 of page 1, system1 are actually continued on system2 of page 3 by staffs 2,3,4 because there is an additional staff 1beginning on that page.

I use the current development branch.

**Comment by hbitteur** [comment]
@phdm 
For this, you definitely need to pick up the development branch. Because there, we introduced a set of features to let the end-user manage score logical parts, together with their corresponding system physical parts. This is precisely what you need.

The related documentation is stilll to be written in Audiveris handbook (I'm working on these doc tasks right now).
Meanwhile, you should have a look at issue #608 (*re-arrange notation lines in complex scores*) I discussed with @Bacchushlg, There you should get enough information on how it works. Please have a look and tell us if you need more information.

**Comment by phdm** [comment]
I opened the LogicalParts Editor on many staffs on many pages, and it opens up empty, except for a colum of 5 5 5,5 5 5 5  in the Counts column.

I also opened up the 'Part#N'/Assign Logical menu but it only shows a white rectangle with a black dot.  I don't really see what I could do there.

I tried to give a name to all the lines in the LogicalParts editor and Save them, but they not appear afterwards in the 'Assign Logical' pop-up, and do not re-appear when I open a second time that LogicalParts editor.

So in summary, I did not succeed to do something useful with those menus.

**Comment by hbitteur** [comment]
@phdm 
Audiveris 5.3 was just released, with significant updates on user handbook.
You should now be more comfortable with the management of logical parts.
Give it a try.
